openeo/rest/datacube.py

```diff
@@ -1881,6 +1881,24 @@ def predict_random_forest(self, model: Union[str, RESTJob, MlModel], dimension:
         reducer = lambda data, context: predict_random_forest(data=data, model=context)
         return self.reduce_dimension(dimension=dimension, reducer=reducer, context=model)
 
+    @openeo_process(mode="reduce_dimension")
+    def predict_catboost(self, model: Union[str, RESTJob, MlModel], dimension: str = "bands"):
+        """
+        Apply ``reduce_dimension`` process with a `predict_catboost` reducer.
+
+        :param model: a :py:class:`MlModel` (e.g. loaded from :py:meth:`~openeo.Connection.load_ml_model`)
+            or a reference to such a model: a URL (``str``) to a STAC Item (implementing the `ml-model` extension)
+            a job id (``str``) or a :py:class:`RESTJob` instance of a batch job that saved a single model.
```

Comment:
see  latest doc of predict_random_forest  for a cleaner parameter description:
https://github.com/Open-EO/openeo-python-client/blob/6f0a86517389c4b3b48309c9c2c37f675b714587/openeo/rest/datacube.py#L1870-L1876